lighthouse-core/test/audits/is-on-https-test.js

```diff
@@ -55,6 +57,24 @@ describe('Security: HTTPS audit', () => {
     });
   });
 
+  it('augmented with mixed-content InspectorIssues', async () => {
+    const networkRecords = [
+      {url: 'https://google.com/', parsedURL: {scheme: 'https', host: 'google.com'}},
+      {url: 'http://localhost/image.jpeg', parsedURL: {scheme: 'http', host: 'localhost'}},
+      {url: 'https://google.com/', parsedURL: {scheme: 'https', host: 'google.com'}},
+    ];
+    const mixedContentIssues = [
+      {insecureURL: 'http://localhost/image.jpeg', resolutionStatus: 'MixedContentBlocked'},
+    ];
+    const artifacts = getArtifacts(networkRecords, mixedContentIssues);
+    const result = await Audit.audit(artifacts, {computedCache: new Map()});
+
+    expect(result.details.headings).toHaveLength(2);
+    expect(result.details.items[0]).toMatchObject({url: 'http://localhost/image.jpeg'});
+    expect(result.details.items[0].resolution).toBeDisplayString('Blocked');
```

Comment:
super-nit: can be fancy and do
```suggestion
    expect(result.details.items[0]).toMatchObject({
      url: 'http://localhost/image.jpeg',
      resolution: expect.toBeDisplayString('Blocked'),
    });
```
(or it could be `s/toMatchObject/toEqual` if both are there?)

Reply:
oh i missed the fancy

neat